### ISSUE TITLE
[Usable-Split Edit button]: There is no announcement after activating 'Split edit button'.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#2046]
**Actual Result:**
There is no announcement if user select 'split edit button' and also, no other option to go back to previous view mode.

**Expected Result:**
For better user experience, presentation view should be available so that user can share back to previous mode and also there should be proper information should be available so that user is able to know.

**Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to the Bot explorer and select it
3.Navigate to 'Split Edit button' and select it
4. Observe the issue

**User Impact:**
Once the user activate 'Split Edit' button it is found that user is not able to change the presentation mode and also no information is announced for the same that view has been changes.

**Test Environment:**
Platform: Win32
Application: Bot Framework V4 Emulator (Version 4.7.1)
OS Build: Version 2004 (OS Build 19541.1000)
Tool: NVDA

**Observation:** Same issue repro on Mac platform
Platform: Mac 10.15.3 Beta (19D49f)
App version: Version 4.7.0-99577 (4.7.0-99577.99577)
Tool: Voice Over

**Guideline Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/doc2.aspx?sourcedoc=%7BF98F4235-B6BD-47D3-8DA4-F28B9B6FBFA8%7D&file=Guideline%20101%20%E2%80%93%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties.docx&action=default&mobileredirect=true&cid=4834e2e9-bdd6-4597-ae14-5a1ccc99ca61

**Attachment:**
[Usable-Split edit button.zip](https://github.com/microsoft/BotFramework-Emulator/files/4063472/Usable-Split.edit.button.zip)

**Comment by v-rampul** [comment]
#HCL-BotFrameworkEmulator; #Emulator-MAC; #HCL; #A11ySev4;
